cli/remotebazel/remotebazel.go

```diff
@@ -519,6 +521,22 @@ func Run(ctx context.Context, opts RunOpts, repoConfig *RepoConfig) (int, error)
 		}
 	}
 
+	envVars := make(map[string]string, 0)
+	for _, envVar := range *envInput {
+		// If a value was explicitly passed in, use that
+		keyValArr := strings.Split(envVar, "=")
```

Comment:
nit: handle values containing "="

```suggestion
		keyValArr := strings.SplitN(envVar, "=", 2)
```